src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -51,7 +53,8 @@ internal class SourceLocalSymbol : LocalSymbol
             bool allowRefKind,
             TypeSyntax typeSyntax,
             SyntaxToken identifierToken,
-            LocalDeclarationKind declarationKind)
+            LocalDeclarationKind declarationKind,
+            bool scoped)
```

Comment:
I'm concerned we are approaching a context where *scoped* is confusing. It feels like a local variable is always *scoped* in a significant sense--is there anything we can do to be more specific/clear about e.g. "scope of referenced variables", for example, here?

Reply:
I agree it was confusing. But after renaming this to `scopedModifier` based on your other comment, it seems clear this is about the modifier rather than a general statement about the local.